forte/data/readers/base_reader.py

```diff
@@ -29,12 +30,12 @@
 from forte.data.multi_pack import MultiPack
 from forte.pipeline_component import PipelineComponent
 from forte.process_manager import ProcessManager
-from forte.utils import get_full_module_name
+from forte.utils.utils import get_full_module_name
```

Comment:
Why did we make `get_full_module_name` go one-level deeper?

Reply:
We made a `utils` folder and moved `utils.py` into that folder.

Reply:
Yeah, I am wondering if we can pull it up by one level through `__init__.py`. `forte.utils.utils` is a bit verbose.

Reply:
When I import something in our repo, I try to import it from its own file so that cycling import won't be triggered.

Reply:
I see. If this could trigger cyclic imports, we can leave it as is.

Reply:
I do not think this one could trigger cyclic imports, but I think this is a good import style when we import something in our repo.